client/components/Assignments/AssignmentPreviewContainer/AssignmentPreviewHeader.tsx

```diff
@@ -70,105 +71,109 @@ export const AssignmentPreviewHeader = ({
                 }
             </ContentBlock>
             <Tools className="AssignmentPreview__toolbar" topTools={true}>
-                <div>
-                    <Item noBg={true} noHover={true}>
-                        {
-                            hideAvatar === true ? null : (
-                                <Column border={false}>
-                                    <UserAvatar
-                                        user={assignedUser}
-                                        size="large"
-                                    />
-                                </Column>
-                            )
-                        }
+                <Spacer h gap="4" justifyContent="start" noWrap>
+                    {hideAvatar !== true && (
                         <Column border={false}>
-                            <Row margin={false}>
-                                <span className="sd-list-item__normal">
-                                    {gettext('Desk:')}
-                                </span>
-                                <span className="sd-list-item__strong">
-                                    {assignedDeskName}
-                                </span>
-                            </Row>
-                            <Row margin={false}>
-                                <span className="sd-list-item__text-label sd-list-item__text-label--normal">
-                                    {deskAssignor && (
-                                        <span>
-                                            {gettext('Assigned by {{name}}', {name: deskAssignorName})},
+                            <UserAvatar
+                                user={assignedUser}
+                                size="large"
+                            />
+                        </Column>
+                    )}
+                    <Spacer v gap="4" noWrap>
+                        <Spacer gap="4" h justifyContent="start" noWrap>
+                            {gettext('Id: {{ assignmentId}}', {assignmentId: assignment._id})}
```

Comment:
1. there's an extra space before `assignmentId`
2. use a generic placeholder like `{{id}}` or `{{value}}` so you don't generate a new translation string for every field.